packages/venia-concept/src/components/MiniCart/__mocks__/body.js

```diff
@@ -0,0 +1 @@
+export default () => '( Body Component Here )';
```

Comment:
You'll notice that there are mocks for each of these internal components that look like this.

This is so that we isolate assertions to the unit (module) under test, not any of its children.

The stories are more of an integration test and can be used as extra verification that the module behaves as expected.

Reply:
This sounds like the prime use case for shallow rendering, which I dabbled in on my re-factor as well (using `react-test-renderer` instead of `enzyme` too). Would that solve the problem you're trying to avoid with these mocks?

eg. https://github.com/magento-research/pwa-studio/blob/96f54750b7701764a4edc2f3506494400dae605a/packages/venia-concept/src/RootComponents/Category/__tests__/categoryContent.spec.js

Reply:
> using react-test-renderer instead of enzyme too

This was the big goal, yeah. I'll check it out, thanks!

Reply:
Umm yeah that's perfect 😄

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1266/commits/735992ca81b59c773a05529f35d67695ed5ff4e3.